[2022-12-04 14:57:22,739] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-08T23:50:00+00:00 [queued]>
[2022-12-04 14:57:22,763] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-08T23:50:00+00:00 [queued]>
[2022-12-04 14:57:22,764] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 14:57:22,764] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-12-04 14:57:22,765] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-04 14:57:22,791] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-11-08 23:50:00+00:00
[2022-12-04 14:57:22,798] {standard_task_runner.py:52} INFO - Started process 12575 to run task
[2022-12-04 14:57:22,803] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'local_to_gcs_task', 'scheduled__2022-11-08T23:50:00+00:00', '--job-id', '9338', '--raw', '--subdir', 'DAGS_FOLDER/AirflowClean2.py', '--cfg-path', '/tmp/tmph1sqzly9', '--error-file', '/tmp/tmp6_p0jpu9']
[2022-12-04 14:57:22,804] {standard_task_runner.py:77} INFO - Job 9338: Subtask local_to_gcs_task
[2022-12-04 14:57:22,878] {logging_mixin.py:109} INFO - Running <TaskInstance: dag1.local_to_gcs_task scheduled__2022-11-08T23:50:00+00:00 [running]> on host 7a020590adae
[2022-12-04 14:57:22,956] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Fadlil
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-11-08T23:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-08T23:50:00+00:00
[2022-12-04 14:57:24,654] {python.py:175} INFO - Done. Returned value was: None
[2022-12-04 14:57:25,000] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=local_to_gcs_task, execution_date=20221108T235000, start_date=20221204T145722, end_date=20221204T145725
[2022-12-04 14:57:25,077] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-04 14:57:25,182] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
